code/game/objects/items/roulette.dm

```diff
@@ -0,0 +1,72 @@
+#define SPINNING_DURATION 3 SECONDS
+
+/obj/structure/roulette
+	name = "roulette"
+	desc = "Normally used for giving out rewards.. or choosing torture methods."
+	icon = 'icons/obj/roulette.dmi'
+	icon_state = "blue"
+	density = TRUE
+	/// Is it currently spinning?
+	var/spinning = FALSE
+	/// List of options the roulette currently has
+	var/list/options = list("10 credits", "20 credits", "50 credits", "100 credits", "200 credits", "500 credits")
```

Comment:
```suggestion
	var/list/options = list("Blue", "Orange", "Green", "Yellow", "Red", "Purple")
```
Think it might be better to have this as the initial options instead of random credit values.